src/test/java/teammates/test/driver/EmailAccount.java

```diff
@@ -152,18 +163,30 @@ private static Credential getCredentialFromFlow(GoogleAuthorizationCodeFlow flow
                 flow, new LocalServerReceiver()).authorize("user");
     }
 
+    /**
+     * Prints the reason why the logged in user is forbidden to access the actual user's emails and what needs to be done.
+     * @param message the message explaining why the logged in user is forbidden to access the actual user
+     * @param username the actual user whose emails are to be accessed
+     */
+    private static void printForbiddenUserMessage(String message, String username) throws IOException {
+        System.err.println(message);
+        System.err.println("Please delete "
+                + new File(TestProperties.TEST_GMAIL_API_FOLDER, username).getCanonicalPath());
```

Comment:
Asking the user to manually delete this is painful; do check out my patch above for a way to do this automatically.